modules/virt-viewing-downward-metrics-cli.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-exposing-downward-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-viewing-downward-metrics-cli_{context}"]
+= Viewing downward metrics by using the command line
+
+You can view downward metrics by entering a command from inside a guest virtual machine (VM).
+
+.Procedure
+
+* Run the following command:
++
+[source,terminal]
+----
+$ sudo sh -c 'printf "GET /metrics/XML\n\n" > /dev/virtio-ports/org.github.vhostmd.1'
+$ sudo cat /dev/virtio-ports/org.github.vhostmd.1
+----
```

Comment:
Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#single-command-per-code-block), we should only have one command per code block in order for the copy/paste button on the website to work. You can still keep these two commands in the same step, but they just need to be in their own blocks:
```suggestion
[source,terminal]
----
$ sudo sh -c 'printf "GET /metrics/XML\n\n" > /dev/virtio-ports/org.github.vhostmd.1'
----
+
[source,terminal]
----
$ sudo cat /dev/virtio-ports/org.github.vhostmd.1
----
```